dockerImages/rabbitmq/Dockerfile

```diff
@@ -2,22 +2,21 @@
 # SPDX-License-Identifier: BSD-2-Clause
 FROM namespace/weathervane-centos7:version
 MAINTAINER hrosenbe@vmware.com
-	
-COPY erlang.repo /etc/yum.repos.d/erlang.repo
+
 COPY erlang.cookie /root/.erlang.cookie
 COPY erlang.cookie /var/lib/rabbitmq/.erlang.cookie
 COPY entrypoint.sh /entrypoint.sh
 COPY rabbitmqadmin  /usr/local/bin/rabbitmqadmin
-COPY isUp.pl /isUp.pl    
+COPY isUp.pl /isUp.pl
 ARG http_proxy
 
 RUN chmod +x /entrypoint.sh && \
 	chmod +x /usr/local/bin/rabbitmqadmin && \
 	chmod 600 /var/lib/rabbitmq/.erlang.cookie && \
     rpm --import https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc && \
-	yum install -y erlang && \
+	yum install -y https://github.com/rabbitmq/erlang-rpm/releases/download/v20.1.7.1/erlang-20.1.7.1-1.el7.centos.x86_64.rpm  && \
```

Comment:
Please change this line to:

`yum install -y https://github.com/rabbitmq/erlang-rpm/releases/download/v21.3.8.21/erlang-21.3.8.21-1.el7.x86_64.rpm && \`

This gives the same version of erlang as was previously used.  The version that you are using here is too old for the version of RabbitMQ.